bzt/engine.py

```diff
@@ -1034,15 +1034,18 @@ def get_headers(self):
         headers = scenario.get("headers")
         return headers if headers else {}
 
-    def get_requests(self):
+    def get_requests(self, require_url=True):
```

Comment:
This strikes me as suspicious.

Maybe we can declare a `Scenario` descendant in `selenium.py` and override this method or something like that?